raspiraw.c

```diff
@@ -575,6 +575,10 @@ static void buffers_to_rawcam(RASPIRAW_CALLBACK_T *dev)
 
 static void callback(MMAL_PORT_T *port, MMAL_BUFFER_HEADER_T *buffer)
 {
+	if ((buffer->user_data == NULL) || (buffer->length == 0))
+	{
+		return;
```

Comment:
Do you insist on the style changes?

I'll make the functional changes.

Reply:
No I don't insist on the style changes for using ```!thing``` instead of ```thing != 0```.

As I put in my summary, MMAL is being stripped out of here anyway!

Reply:
Done.